.github/workflows/sc-docker-images-bldr.yml

```diff
@@ -0,0 +1,111 @@
+name: sc-docker-images-bldr
+
+on:
+  push:
+    branches: [ "**" ]
+    paths:
+      - '.github/workflows/sc-docker-images-bldr.yml'
+      - 'dockerfiles/Dockerfile'
```

Comment:
Are all these Dockerfiles dependencies of the build instructions executed in this CI workflow? If not, then it's unnecessary (and somewhat misleading) to trigger a build if a non-dependent file changes. It might be better to have a trigger and build for each [group of] Dockerfile if it needs to be tested.

Reply:
I agree. I will think how to do it better. Thank you!